nixops/deployment.py

```diff
@@ -1269,7 +1270,7 @@ def _create_state(depl, type, name, id):
 def _load_modules_from(dir):
     for module in os.listdir(os.path.dirname(__file__) + "/" + dir):
         if module[-3:] != '.py' or module == "__init__.py": continue
-        __import__("nixops." + dir + "." + module[:-3], globals(), locals())
+        importlib.import_module("nixops." + dir + "." + module[:-3])
```

Comment:
revert?

Reply:
I would recommend `importlib` over `__import__`. Since the latter one does not guarantee portability across python versions: 
https://docs.python.org/3/library/importlib.html#introduction

Reply:
also see: https://docs.python.org/2.7/library/importlib.html#introduction